### PR TITLE
Update test files to conform to eslint rules.

### PR DESCRIPTION
**Pull request by eschlange**
* Updated spec files to use eslint rules.
* Removed extranious log files in git ignore.

[#104362210]

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -13,6 +13,7 @@
     "quotes": [2, "single"],
     "no-eval": 0,
     "no-unused-vars": 1,
-    "no-multiple-empty-lines": [2, {"max": 2}]
+    "no-multiple-empty-lines": [2, {"max": 2}],
+    "global-strict": [0]
```
Comment (ericcf) [inline .eslintrc]: Looks like this rule is deprecated: http://eslint.org/docs/rules/global-strict

Otherwise, any reason for turning off strict mode?

**Comment by ericcf** [comment]
Looks cleaner to me!

**Comment by eschlange** [comment]
@ericcf that eslint strict rule removal was an accident. I updated the config to include a rule that specifically checks for it.

**Comment by ericcf** [comment]
:panda_face:

**Pull request merged**